build.rs

```diff
@@ -0,0 +1,98 @@
+use html5ever::parse_document;
+use html5ever::tendril::TendrilSink;
+use markup5ever_rcdom::{Handle, NodeData, RcDom};
+use std::{fs, io::Write, path::Path};
+
+const LINK_RELATIONS_PAGE: &str =
+    "https://www.iana.org/assignments/link-relations/link-relations.xhtml";
+
+fn main() {
+    println!("main");
+
+    let client = reqwest::blocking::Client::new();
+    let html = client
+        .get(LINK_RELATIONS_PAGE)
+        .header(
+            reqwest::header::USER_AGENT,
+            "Mozilla/5.0 (compatible; CopilotBot/1.0)",
+        )
+        .send()
+        .expect("Failed to fetch page")
+        .text()
+        .expect("Failed to read response text");
```

Comment:
We can't do that. This would break ammonia in build environments like docs.rs that don't have internet access.